docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -631,6 +649,14 @@ scenario, and input draw.
     - Same as 4.4
     - AD-dementia incidence looks identical to 4.4, so the double rounding was perhaps not a problem after all
     - https://github.com/ihmeuw/vivarium_research_alzheimers/blob/9fef98e6cc61f4eb6dec96c2cd477e64cc084d3f/verification_and_validation/2025_09_18d_model4.5_vv.ipynb
+  * - 6.0
+    - * Only eligible simulants are tested based on :ref:`PET/CSF <petcsf_requirements>` and :ref:`BBBM <bbbm_requirements>` testing requirements.
+      * Location-specific CSF vs PET testing rates (CSF tests / PET tests = CSF rate / PET rate)
+      * 90% diagnostic rate for BBBM tests
```

Comment:
I think "diagnostic rate" sounds too imprecise. Does this mean that the test should have a 90% sensitivity (true positive rate), or equivalently, a 10% false negative rate?